Packages/Tracking/Hands/Runtime/Scripts/HandBinder/HandBinder.cs

```diff
@@ -272,7 +272,7 @@ void TransformElbow()
                 Vector3 dir = -LeapHand.Arm.Direction.normalized;
 
                 //Position the elbow at the models elbow length
-                Vector3 position = LeapHand.WristPosition + dir * (ElbowLength);
+                Vector3 position = LeapHand.WristPosition + dir * (ElbowLength * transform.localScale.z);
```

Comment:
I think these `transform.localscale.z` could be more specific in relation to the elbow by using 
`BoundHand.elbow.boundTransform.lossyScale.z `